b2b-sample/libs/business-admin-app/data-access/data-access-controller/src/lib/controller/identityProvider/controllerCreateIdentityProvider/controllerDecodeCreateIdentityProvider.ts

```diff
@@ -16,11 +16,27 @@
  * under the License.
  */
 
-import { IdentityProvider, IdentityProviderTemplate, IdentityProviderTemplateModel, setIdpTemplate } from
+import {
+    IdentityProvider, IdentityProviderConfigureType, IdentityProviderDiscoveryUrl, IdentityProviderTemplate,
+    IdentityProviderTemplateModel, setIdpTemplate
+} from
     "@b2bsample/business-admin-app/data-access/data-access-common-models-util";
 import { commonControllerDecode } from "@b2bsample/shared/data-access/data-access-common-api-util";
 import { Session } from "next-auth";
 import { controllerCallCreateIdentityProvider } from "./controllerCallCreateIdentityProvider";
+import controllerDecodeGetDiscoveryUrl from "../controllerGetDiscoveryUrl/controllerDecodeGetDiscoveryUrl";
+
+/**
+ * get the identity provider details from the discovery url
```

Comment:
`get the identity provider details from the discovery url` ->
`Get the identity provider details from the discovery url.`

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/c9fb90d7d7ab19441f2e0d135cf2a6f218b7c78b